### DISCUSSION TITLE
Create a HH-60J

### DISCUSSION THREAD

**Discussion by PatrickThompson69** [Project-Hatchet/public_h-60#352] [Ideas]
I think this would be a very cool to add to this mod would be super realistic

**Comment by Riverman02** [comment]
Only way this is happening is if someone provides a model for us (legally) and, assuming we would even have the time, incorporating the interaction and the rest of the systems in a realistic fashion (there are significant differences on the interior) on an aircraft that is ultimately an edge-case is unlikely to happen unless significant amounts of money start getting involved 😂

**Reply by PatrickThompson69** [reply]
I have a model

**Reply by PDTFawks** [reply]
"provides a model for us"
To be specific, a high quality, Arma ready model with the animations already done.
Even given that it would be a team vote on if we do it. Our current birds still need quite a bit of love as is so another bird is not likely at the moment. Lost and BroBeans may have more input than I though.
